classes/observer.php

```diff
@@ -173,6 +173,9 @@ public static function email_mention_hsu(\mod_hsuforum\event\assessable_uploaded
                         $post_id = $event->objectid;
                         $course_id = $event->courseid;
 
+                        //@all already here strip href from @all tag, leave for everything else
+                        $content = preg_replace('#<a.*?>(.*?)@all</a>#i', '@all', $content);
```

Comment:
Any reason for not using strip_tags? If strip_tags works, would prefer that over a custom regex